enhancements/etcd/cluster-etcd-operator.md

```diff
@@ -180,6 +180,16 @@ This controller shapes the static pod manifest itself.  It has some unique featu
     It waits until it is able to know that it is in the member list before trying to start.
     When it does this, it is able to determine the member list to use to launch.
 
+### Replacing single lost master
+This is one of the scenarios that needs user intervention and the step needs to be in user-facing docs.
+
+1. User will determine if one of the etcd is unhealthy by verifying against etcd API from etcd pod: `etcdctl endpoint health --cluster`
```

Comment:
I guess one option is that "repair a broken etcd" is done by creating a new master and removing the old.  that collapses state nicely.